Makefile

```diff
@@ -57,3 +57,9 @@ lint-fix: bin/golangci-lint ## Fix lint violations.
 .PHONY: generate
 generate: ## Run all generate-* jobs in bulk.
 	cd tools; go generate
+
+##@ New kubernetes control-plane patchversions
```

Comment:
I think this name is too specific. It sounds like a group for a single target to me. I would not say this blocks us from merging the PR, but any movements will be good.